packages/react/src/components/Flex/Flex.tsx

```diff
@@ -28,7 +28,7 @@ export interface FlexProps {
   space?: 'around' | 'between' | 'evenly'
 
   /** Defines gap between each two adjacent child items. */
-  gap?: 'gap.small' | 'gap.medium' | 'gap.large'
+  gap?: 'gap.small' | 'gap.medium' | 'gap.large' | 'gap.smaller'
```

Comment:
NIT: we can order by size:` 'gap.smaller' | 'gap.small' | 'gap.medium' | 'gap.large'`